Package.swift

```diff
@@ -60,11 +60,25 @@ let package = Package(
         .target(
             name: "AutomatticTracksEvents",
             dependencies: ["AutomatticTracksModel",
-                           "AutomatticExperiments"],
+                           "AutomatticExperiments",
+                           "AutomatticTracksEventsForSwift"],
```

Comment:
Since Swift Packages don't support mixed-source, I've created a Swift module to open a path to incrementally migrate module `AutomatticTracksEvents` to Swift (`AutomatticTracksEventsForSwift`).

The idea is that once that migration is complete, it should be possible to simply remove the old module and rename `AutomatticTracksEventsForSwift` > `AutomatticTracksEvents`.